docs/guides/studio_oauth.rst

```diff
@@ -65,3 +65,11 @@ Config and code changes to be performed after all environments are using OAuth f
 - Remove ``LOGIN_URL`` overrides from all environments (devstack and others)
 - Remove remaining ``ARCH-1253`` detritus (login redirect)
 - Remove this doc!
+
+Declining the migration
+-----------------------
```

Comment:
We don't know if anyone will want to do this, but I'm just keeping notes here so we keep this in mind for release notes.